docs/RollingFile.md

```diff
@@ -0,0 +1,167 @@
+# Rolling File
+
+In this tutorial, we will be building a console application that does basic rolling file logging.
+
+---
```

Comment:
Same as above 😄

Reply:
Yeah, it does look better with those removed.